audiorecorder/src/main/java/org/odk/collect/audiorecorder/mediarecorder/MediaRecorderRecordingResource.kt

```diff
@@ -61,3 +44,25 @@ internal class MediaRecorderRecordingResource(private val mediaRecorder: MediaRe
         return mediaRecorder.maxAmplitude
     }
 }
+
+internal class AACRecordingResource(mediaRecorder: MediaRecorder) : MediaRecorderRecordingResource(mediaRecorder) {
```

Comment:
Pulling these out was a bit of a distraction, but I was focusing on cleaning the interface up a bit and it made sense for the codec config to live separately from the mechanics of recording. This hopefully makes it easier to add a new implementation or codec.